src/main/java/heylo/Main.java

```diff
@@ -0,0 +1,45 @@
+package heylo;
+
+import heylo.commands.Command;
```

Comment:
Suggestion: The Checkstyle plugin for Intellij orders standard java package imports before other imports. It may be useful to follow this standard so that your import ordering can be standardized with your teammates for tp 😄